lib/metadataTypes/MobileCode.js

```diff
@@ -26,7 +30,7 @@ class MobileCode extends MetadataType {
      * @returns {Promise.<TYPE.MetadataTypeMapObj>} Promise of metadata
      */
     static retrieveForCache() {
-        return super.retrieveREST(null, '/legacy/v1/beta/mobile/code/', null);
+        return super.retrieveREST(null, '/legacy/v1/beta/mobile/code/');
```

Comment:
same WHERE clause logic applies to mobile Code AND mobile keywords :)

Reply:
ok, i can update the one thats using the collection filter based on that :) can you do the verification please? couldn't find a test environment

Reply:
done - please **retest this Code and Keyword on your server**! @DougMidgley